Orange/preprocess/impute.py

```diff
@@ -207,6 +207,14 @@ def __call__(self, data):
         column[mask] = predicted
         return column
 
+    def __eq__(self, other):
```

Comment:
Well, as long as it does not crash it is probably "fine". :) I did fix another bug while playing with this: `__hash__` was wrong because superclass (Reprable) `__hash__` does something strange.

Reply:
Objects are equal if they are same.

```
>>> a = A()
>>> b = A()
>>> a == b
False
>>> a == a
True
```

In case of models, this suffices. Equality of models (beyond sameness) could be difficult to check, so let's not go there. Note that Python's functions behave the same way.

```
>>> (lambda: 42) == (lambda: 42)
False
```